### PR TITLE
Fixing issues raised by the linter, testing CI

### PR DESCRIPTION
**Pull request by miselico**

### CODE REVIEW

File: assessment/assessment.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ func (inst *Instance) GetWikiRecs(Properties []string) schematree.PropertyRecomm
 
 	var res *http.Response
 	var err error
-	for true { // retry like a maniac
+	for { // retry like a maniac
```
Comment (reedy) [inline assessment/assessment.go:91]: maniac is not a good choice of words

Comment (miselico) [reply assessment/assessment.go:91]: Fixing in the branch. I made the pull request to test the CI. I will close this one now and create a new one. that should also trigger gosec.

Comment (miselico) [reply assessment/assessment.go]: Apparently, pushing the change triggers gosec as well. Keeping this pull request open. We will merge it once we have also the remaining linting/gosec issues fixed.

Comment (miselico) [reply assessment/assessment.go]: fixed in ab9661b

**Pull request merged**